hieradata/vagrant/common.yaml

```diff
@@ -388,12 +388,12 @@ profile::jenkinscontroller::plugins:
   - workflow-multibranch
 profile::buildagent::aws_credentials: |
   [default]
-  aws_access_key_id=AKIAIOSFODNN7EXAMPLE
-  aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  aws_access_key_id=EXAMPLE
```

Comment:
simplify due to an alert on my git on the commit `[ERROR] Matched one or more prohibited patterns`

Reply:
This alert comes from your local git configuration, you received it when trying to commit this file, to alert you about a potential secret leak.

As it's not a secret I would have added `--no-verify` to your git command to override this false alert and kept examples (from AWS) as they were.

Reply:
At the very least, I would chose more explicit values like `AWS_ACCESS_KEY_EXAMPLE`, or `AN_AWS_ACCESS_KEY`

Reply:
I would rather used really fake version instead of using an exception. I had to check if this one was not a real one with EXAMPLE at the end, so loose time ... and moreover, if a tool with the same kind of pattern detection put a warning on it, we will have to loose time to check.